bundles/CoreBundle/EventListener/Frontend/DocumentFallbackListener.php

```diff
@@ -41,23 +41,48 @@ class DocumentFallbackListener implements EventSubscriberInterface
 {
     use PimcoreContextAwareTrait;
 
+    /**
+     * @var RequestStack
+     */
+    protected $requestStack;
+
+    /**
+     * @var DocumentResolver
+     */
+    protected $documentResolver;
+
+    /**
+     * @var SiteResolver
+     */
+    protected $siteResolver;
+
+    /**
+     * @var Document\Service
+     */
+    protected $documentService;
+
     /**
      * @var array
      */
-    protected array $options;
+    protected $options;
 
     /**
      * @var Document|null
      */
-    private ?Document $fallbackDocument;
```

Comment:
`private ?Document $fallbackDocument = null;`

would solve that

Reply:
Thanks!